index.bs

```diff
@@ -165,37 +165,61 @@ by showing how many sites are willing to ask for intrusive and unnecessary permi
 
 <h3 id="naming-is-hard">Naming things</h3>
 
-Naming is hard! We would all like a silver-bullet for naming APIs...
-
 Names take meaning from:
 
 * signposting (the name itself)
 * use (how people come to understand the name over time)
 * context (the object on the left-hand side, for example)
 
-Consistency is a good principle that helps to create a platform that users can navigate intuitively
-and by name association.
+<h4 id="naming-common-words" class="no-num no-toc">Use common words</h4>
 
-Please consult widely on names in your APIs.
+API naming *must* be done in easily readable US English.
+Keep in mind that most web developers are not native English speakers.
+Whenever possible, names should be chosen that use common vocabulary
+non-native English speakers are likely to understand when first encountering the name.
+Always value readability over brevity.
```

Comment:
I'm not sure I entirely agree with this. E.g., how would this judge `fetch()` or `response.json()`?

Reply:
Applying any or all of these rules retrospectively may not necessary work. In your example I'd argue that `fetch()` is a common vocabulary for getting things on one's behalf. On the other hand, `json` is a well known reference similar to `style`, `css` etc. that shouldn't need a long form... if the format isn't known to the user it should be trivial to find of course.

Reply:
Well, it might not be clear that it `json()` consumes the body of the response though. That's more the thing I was getting it. But also, I think that there's value in brevity as it's never possible to encapsulate the full complexity in the name.

Reply:
I've tried to capture this in d59b3a7, @atanassov & @annevk.